library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/StringObfuscatorTest.kt

```diff
@@ -4,11 +4,10 @@
  * Copyright 2016-Present Datadog, Inc.
  */
 
-package com.datadog.android.sessionreplay.internal.recorder.mapper
+package com.datadog.android.sessionreplay.internal.recorder.obfuscator
```

Comment:
should be still `mapper` package according to the location? Or file should be moved?